index.js

```diff
@@ -11,14 +11,21 @@ if (other) {
   }
 } else
 if (osx) {
-  var regPath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  var osxSuffix = '/Contents/MacOS/Google Chrome'
+  var regPath = '/Applications/Google Chrome.app' + osxSuffix
   var altPath = require('userhome')(regPath.slice(1))
+  var mdFindCmd = 'mdfind \'kMDItemDisplayName == "Google Chrome" && kMDItemKind == Application\''
 
-  module.exports = fs.existsSync(regPath)
-    ? regPath
-    : altPath
+  if (fs.existsSync(regPath)) {
+    module.exports = regPath
+  } else if (fs.existsSync(altPath)) {
+    module.exports = altPath
+  } else {
+    var foundPath = require('child_process').execSync(mdFindCmd, { encoding: 'utf8' })
```

Comment:
It would be great to have your input on https://github.com/davidtheclark/safari-location/blob/master/index.js (latest on master is all async, returns promise and calls callback). If I can get that module to your liking, I will apply the same to Chrome, Firefox, and IE.